Nodejs/Common/Telemetry/TelemetryEvents.cs

```diff
@@ -20,5 +20,7 @@ namespace Microsoft.NodejsTools.Telemetry {
     /// </summary>
     internal static class TelemetryEvents {
         public const string ProjectImported = "ProjectImported";
+        public const string IntelliSenseActivatedForProject = "IntelliSenseActivatedForProject";
```

Comment:
Should this be AnalysisLevel instead? We seem to use IntelliSense level or IntelliSense mode in the UI and documentation, but analysis level in the code.